src/openrct2/ride/ride.h

```diff
@@ -912,6 +912,16 @@ enum {
     TRACK_SELECTION_FLAG_RECHECK          = 1 << 3,
 };
 
+enum {
+    SEQUENCE_NUMBER = 0xF,
+    SEQUENCE_10 = 0x10,
+    SEQUENCE_30 = 0x30,
+    SEQUENCE_GET_STATION = 0x70,
+    SEQUENCE_REMOVE_STATION = 0x8F,
+    SEQUENCE_HAS_GREEN_LIGHT = 0x80,
+    SEQUENCE_TAKING_PHOTO = 0xF0,
```

Comment:
These flags aren't related to sequence, so they should be named something else, like `TRACK_VISUAL_FLAG`